adoc/Release-Notes.adoc

```diff
@@ -170,19 +171,20 @@ Delete the PVC(s) for `mysql-1` from both `uaa` and `scf` namespaces but leave t
 [source,bash]
 ----
 helm upgrade susecf-uaa --namespace uaa suse/uaa  -f <values.yaml> --set config.HA=true \ 
---set config.HA_strict=false --set sizing.mysql.count=1
+--set config.HA_strict=false --set sizing.mysql.count=1 --version 2.18.0
 helm upgrade susecf-scf --namespace scf suse/cf -f <values.yaml> --set config.HA=true \ 
---set config.HA_strict=false --set sizing.mysql.count=1 --set "secrets.UAA_CA_CERT=${CA_CERT}"
+--set config.HA_strict=false --set sizing.mysql.count=1 --set "secrets.UAA_CA_CERT=${CA_CERT}" \
```

Comment:
Do we not need to grab the new secret and certificate values again right after we upgraded UAA from 2.17.1 to 2.18.0? I think the secrets-generator would generate new secret and a corresponding `INTERNAL_CA_CERT`...?